erpnext/selling/doctype/customer/customer.json

```diff
@@ -768,11 +779,12 @@
   }
  ],
  "quick_entry": 1,
- "search_fields": "customer_name,customer_group,territory,mobile_no, mobile_no_2, phone_no, email_id, tax_id, tax_cnic, tax_strn",
+ "search_fields": "customer_name,customer_group,territory,mobile_no, mobile_no_2, phone_no, email_id, tax_id, tax_cnic, tax_strn, reference",
```

Comment:
Lets move `reference` after `customer_name` and remove `customer_group` and `territory` from search_fields